src/components/Editor/index.js

```diff
@@ -69,6 +69,16 @@ export default class UnicornEditor extends Component {
   };
 
   render() {
+    const stickerElements = stickers.map((sticker) => (
```

Comment:
Will this ever rerender, should we move this and the `.toArray` in the `return` on line 94 to the constructor?